### ISSUE TITLE
User Registration

### ISSUE THREAD

**Issue by ryanhiebert** [tycoles/mean-chat#4]
Goes along with #3.

Make a way to create new users. Perhaps only give this ability to already existing users, so that its an invite-only thing?

**Comment by alankearney** [comment]
Hi Ryan,
Just noticed you were working on this when I made a commend about staying with AngularJS. I should have looked before I made the comment.  If you already made progress, let's not loss it.

**Comment by ryanhiebert** [comment]
No, I haven't been working on it at all, just opened a couple new issues about what we could do instead of #2, per your comment about wanting to work more with Mongo.
